src/main/java/mate/academy/model/ShoppingCard.java

```diff
@@ -0,0 +1,58 @@
+package mate.academy.model;
+
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToMany;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "shopping_cards")
+public class ShoppingCard {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToMany
+    @JoinColumn(name = "ticket_id")
+    private List<Ticket> tickets;
+    @ManyToMany
```

Comment:
```suggestion
    @OneToOne
```

one Shopping cart could have many users?
also read [checklist](https://mate-academy.github.io/jv-program-common-mistakes/hibernate/add-shopping-cart/add-shopping-cart-hw): Use @MapsId with ShoppingCart. Think about whether to use an annotation @GeneratedValue in ShoppingCart.